http-server/src/main/java/org/triplea/modules/chat/event/processing/PlayerIsMutedMessage.java

```diff
@@ -0,0 +1,26 @@
+package org.triplea.modules.chat.event.processing;
+
+import com.google.common.annotations.VisibleForTesting;
+import java.time.Clock;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.function.BiFunction;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+class PlayerIsMutedMessage {
+
+  @VisibleForTesting
+  static final BiFunction<Clock, Instant, String> muteDurationRemainingToString =
+      (clock, muteExpiry) -> {
+        final long minutes = Duration.between(clock.instant(), muteExpiry).toMinutes();
+        return minutes > 0
+            ? minutes + " minutes"
+            : Duration.between(clock.instant(), muteExpiry).toSeconds() + " seconds";
+      };
```

Comment:
Thoughts on using a simple static method instead of a lambda edxpression?

Reply:
I think this was done to be able to override and mock the function, looks like that relic. I'll look into the test cases, this probably can be a 'simple' method. On the other hand, static `@VisibleForTesting` methods are a code smell, likely better to make this be injected behavior.

Reply:
Improved this a bit by making the bi-function to be a static inner class: b8ef3f663